### ISSUE TITLE
Message jina.RequestProto exceeds maximum protobuf size

### ISSUE THREAD

**Issue by LemurPwned** [jina-ai/finetuner#232]
Hey,
thanks for the great package!
I'm currently testing the fine-tuning for some image detection. 
I'm not passing images to the model but embeddings (I'm training an extension only), so my document looks like this
![image](https://user-images.githubusercontent.com/8441184/143611004-e0cbf236-aa76-4ca2-9a77-646d3e1d5443.png)
(embedding is here just a 128-dim vector). My model is PyTorch 2 linear layers that inference on that embedding and spit out 128 dimensional vector. 

The UI works great but after doing a couple rounds of annotations I repeatedly run into the following error:
![image](https://user-images.githubusercontent.com/8441184/143610845-1eba48ca-a4f8-4e30-8197-ead52d6b1440.png)

Any idea what could be responsible for that error?

**Comment by JoanFM** [comment]
Hey @LemurPwned ,

May u try using a smaller batch size? I think this should mitigate the problem

**Comment by LemurPwned** [comment]
@JoanFM Thanks, that was my thought too -- I tried to reduce it to 4 but I still get the same error. My GPU is 11 GB VRAM so I don't think should pose problems.

**Comment by hanxiao** [comment]
hmm 
- it could be `uri` (data uri) is too big?
- given that it complains about request proto and 2G max, it must be the overall size of all Documents in a Request is too large.

I think its latter, but i'm outside will do a check later

**Comment by LemurPwned** [comment]
@hanxiao 
So regarding data uri being too big --  The `blob` field, as I mentioned is 128-dim float32 vector.
For the images, I've been using the `jina.types.document.png_to_buffer` function as some examples also indicate. However, this may accumulate for longer-runs and multiple images.
![image](https://user-images.githubusercontent.com/8441184/143845540-9e89f809-52d8-4f28-a435-e074ad99d897.png)

**Comment by gmastrapas** [comment]
Hey @LemurPwned! What finetuner version are you using? Could you also share some insight on the dataset size and the number of docs you manage to label before the exception is raised? Also the complete data generator implementation would be helpful
